app/javascript/stylesheets/blocks/_details.scss

```diff
@@ -0,0 +1,26 @@
+.details {
+  @include element('summary') {
+    // trick to avoid the text going under the arrow
+    & > * {
+      display: inline;
```

Comment:
This is beyond my scss skills haha! I think I'm following after a good amount of googling!